documentation/tutorial_siteBranding.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: master
+permalink: /documentation/tutorial_branding/
+title: Tutorial Site Branding
+tutorial: true
+weight: 10
+---
+# Tutorial: Site Branding
+
+### Requirements
+
+* A development environment
+* An Elastic Path training virtual machine
+
+### Prerequisites
+
+Ensure that you are familiar with the following third-party technologies:
+* Git
+* Node.js
+* Visual Studio Code with the following extensions:
+    * Debugger for Chrome
+    * ESLint extension
+
+### Example
+
+**Before you begin**:
+* For device compatibility, ensure that any image names and the resolution are the same.<br>
+
+1. Clone or pull the `react-pwa-reference-storefront` repository to your directory.
+2. Navigate to the `src/ep.config.json` file.
+3. Update the value of `cortexApi.scope` with the store's name.
+4. Add marketing content images in the `react-pwa-reference-storefront⁩/src⁩/images⁩/site-images` file. <br>
+    **Note**: To replace images and keep the same layout in other components, ensure that the image names are the same. <br>
```

Comment:
You can reuse the same images and layout if you do not change image names > is this what you wanted to convey?

Reply:
Yes, and I was having a hard time conveying that. Ideas welcome :)